docs/configuration.rst

```diff
@@ -0,0 +1,60 @@
+exodus-gw configuration
+=======================
+
+Required environment variables
+..............................
+
+All ``pubtools-exodus`` operations require the ``EXODUS_ENABLED`` environment variable to be set
```

Comment:
As mentioned in https://github.com/release-engineering/pubtools-exodus/pull/12 I don't think this should actually be the case.  It's very unusual for a *command* to do nothing by default unless a certain env var is set.

I think the doc & behavior for this option should be something like: pubtools-exodus can optionally extend the behavior of other projects in the pubtools family, causing them to integrate with exodus-gw.  This is enabled if and only if `EXODUS_ENABLED` is set.

And then it might make sense for the header to be something like "Enabling pubtools-exodus hooks" or similar.